.github/workflows/check-licenses.yml

```
@@ -0,0 +1,26 @@
+name: Check licenses
+
+on:
+  pull_request:
+
+jobs:
+  check-licenses:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3.8.1
+        with:
+          node-version: '18.x'
+          registry-url: 'https://npm.pkg.github.com'
+          scope: '@paritytech'
+      - name: Check the licenses
+        run: |
+          shopt -s globstar
+
+          npx @paritytech/license-scanner@0.0.5 scan \
```

```suggestion
          npx --yes https://github.com/paritytech/license-scanner scan \
```

Not ideal, but maybe work as hot-fix.